frontend/pages/hubs/[hubUrl].tsx

```diff
@@ -335,20 +277,13 @@ export default function Hub({
           {!isLocationHub && <BrowseExplainer />}
           <BrowseContext.Provider value={contextValues}>
             <BrowseContent
```

Comment:
removed filters related hooks but added the initalFilters and filterChoices as these might need values provided by SSR